modules/elastodyn/src/ElastoDyn.f90

```diff
@@ -3206,6 +3215,9 @@ SUBROUTINE SetTowerParameters( p, InputFileData, ErrStat, ErrMsg  )
       p%StiffTFA  (J) = InterpStp( p%HNodesNorm(J), InputFileData%HtFract, InputFileData%TwFAStif, InterpInd, InputFileData%NTwInpSt )
       p%StiffTSS  (J) = InterpStp( p%HNodesNorm(J), InputFileData%HtFract, InputFileData%TwSSStif, InterpInd, InputFileData%NTwInpSt )
    END DO ! J
+   p%MassT = abs(p%MassT)
+   p%StiffTFA = abs(p%StiffTFA)
+   p%StiffTSS = abs(p%StiffTSS)
```

Comment:
I think the absolute value might be fine here because we don't have cross coupling stiffnesses. But I think cross-coupling stiffnesses can be "negative". I would probably do something like:
```
if p%MassT <0:
   p%MassT =  -p%MassT
   p%StiffTFA = -p%StiffTFA
   p%StiffTSS = -p%StiffTSS
endif
```
Maybe @jjonkman can comment on this.

Reply:
I'm familiar with this PR and am OK with the absolute value for now.  I agree that a possible future "general ElastoDyn" would require a more rigorous approach.